invenio_communities/subcommunities/services/service.py

```diff
@@ -51,6 +52,19 @@ def expandable_fields(self):
         """Get expandable fields."""
         return []
 
+    def _is_owner_of(self, identity, community):
+        """Check if the identity provides community ownership."""
+        return next(
```

Comment:
I considered the following ways of checking for ownership:

1. inspecting the identity (current implementation)
2. checking the memberships of the user and look for one that is `owner`

If you have any other ideas, please share